src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -26,13 +28,18 @@
 import java.util.concurrent.TimeoutException;
 import java.util.concurrent.atomic.AtomicReference;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class ConsumerTest extends HttpBridgeTestBase {
 
+    private static final Logger LOGGER = LogManager.getLogger(ConsumerTest.class);
```

Comment:
what about putting the logger in the base class?

Reply:
Yeah, good idea. I will `create task in JIRA and issue` for that and after release i will take a look on that. I moved the `logger` to base class.

Reply:
[Issue migrate](https://github.com/strimzi/strimzi-kafka-bridge/issues/352)